sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -63,7 +63,7 @@ class DefaultAzureCredential(ChainedTokenCredential):
     """
 
     def __init__(self, **kwargs):
-        authority = kwargs.pop("authority", None) or KnownAuthorities.AZURE_PUBLIC_CLOUD
+        authority = os.environ.get(EnvironmentVariables.AZURE_AUTHORITY_HOST) or KnownAuthorities.AZURE_PUBLIC_CLOUD
```

Comment:
This is a breaking change. We must continue to accept `authority` as a keyword argument. I think this is the best order for getting the value:
1. keyword argument: `authority`
1. environment: `AZURE_AUTHORITY_HOST`
1. default: `KnownAuthorities.AZURE_PUBLIC_CLOUD`

Reply:
Absolutely.  Did not intend for the param to be removed. Apologies for that oversight.